pkgs/by-name/za/zapzap/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zapzap";
+  version = "4.5.5.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "zapzap-linux";
+    repo = "zapzap";
+    rev = version;
+    hash = "sha256-8IeFGTI+5kbeFGqH5DpHCY8pqzGhE48hPCEIKIe7jAM=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
```

```suggestion
    setuptools
    qt6.wrapQtAppsHook
```
this and the others fix startup on wayland